README.md

```diff
@@ -23,33 +24,18 @@ More information:
 
 ## Contents
 
-* [What's new in Zowe Explorer 1.9.0](#what's-new-in-zowe-explorer-1.9.0)
+* [What's new in Zowe Explorer 1.10.0](#what's-new-in-zowe-explorer-1.10.0)
 * [Prerequisites](#prerequisites)
 * [Getting Started](#getting-started)
 * [Credentials Security](#credentials-security)
 * [Usage Tips](#usage-tips)
 * [Extending Zowe Explorer](#extending-zowe-explorer)
 
-## What's new in Zowe Explorer 1.9.0
+## What's new in Zowe Explorer 1.10.0
 
 New features:
 
-* Added the Allocate Like feature.
-* Added the ability to disable/enable profile validation.
-* Added the ability to access other profiles during profile validation.
-
-Enhancements:
-
-* Datasets and members names are displayed in all uppercase letters even if typed in lowercase letters.
-* Grouped Favorites by profile for Datasets, USS, and Jobs.
-* Other profiles can be accessed during profile validation.
-* Updated the environment check for Theia compatibility.
-
-Bug Fixes:
-
-* Fixed USS renaming issues.
-* Fixed the deletion of datasets issue.
-* Removed errors in Favorites items caused by profiles that are created by other extensions.
+* Added base profile support. You can now log in to the API Mediation Layer via Zowe CLI and use that token with a service profile in Zowe Explorer.  
```

Comment:
@zdmullen I kind of don't want to use login because that would be the next feature that we are implementing. What do you think? Connecting would be nice but I am not sure

Reply:
I guess I thought that the "via Zowe CLI" covers the current behavior--I know log in from Zowe Explorer comes next. Does that make sense?

Reply:
that sounds good

Reply:
What does "that token" in "and use that token with a service profile mean"?